src/services/auth.service.js

```diff
@@ -79,3 +84,34 @@ function register(name, email, phone, username, password1, password2) {
             }
         });
 }
+function chechUser(userId) {
+    console.log(userId);
+     return axios.get(
+        stringFormat(`${config.commonConfig.$apiUrl}/${config.userConfig.api.checkUser.endpoint}`, userId)
+    )
+
+        .then((response) =>{
+            console.log('response =', response.data);
+        })
+        .catch((error) => {
+            console.log('error =', error.data)
```

Comment:
`throw 'Provided Username / Email does not exist.'`
- Move the string into configuration error messages.